packages/docs/celo-gold-holder-guide/voting-validators.md

```diff
@@ -0,0 +1,81 @@
+# Voting on Validator Groups
+
+## Proof of Stake Election Essentials
+
+Validators play a critical role in the Celo protocol, determining which transactions get applied and producing new blocks. Selecting organizations that operate well-run infrastructure to perform this role effectively is essential for Celo's long-term success.
+
+The Celo community makes these decisions through elections for [Validator Groups](../celo-codebase/protocol/proof-of-stake/validator-groups.md), intermediaries between voters and validators. Every validator group has an ordered list of up to 5 candidate validators. Some organizations may operate a group with their own validators in it; some may operate a group to which they have added validators run by others.
+
+[Validator elections](../celo-codebase/protocol/proof-of-stake/validator-elections.md) are held every epoch (approximately once per day). All 100 slots are available at each election. Validators are selected [in proportion](../celo-codebase/protocol/proof-of-stake/validator-elections.md#Running-the-Election) to votes received for each Validator Group.
+
+If you hold Celo Gold (or are a beneficiary of a [`ReleaseGold` contract](release-gold.md) that allows voting), you can vote for Validator Groups. A single account can split their LockedGold balance to have outstanding votes for up to 10 groups.
+
+Celo Gold that you lock and use to vote for a group that elects one or more validators receives [epoch rewards](../celo-codebase/protocol/proof-of-stake/epoch-rewards.md) every epoch (approximately every day) once the community passes a governance proposal enabling rewards. The initial level of rewards is anticipated to be around 6% per annum equivalent (but is subject to change).
+
+Unlike a number of Proof of Stake protocols, Celo Gold used for voting is never at risk. The actions of the validator groups or validators you vote for can cause you to receive lower or higher rewards, but the Celo Gold you locked will always be available to be unlocked in the future.
```

Comment:
I recently realized the most correct way to write it is "proof-of-stake" and standardized it elsewhere in the docs
```suggestion
Unlike a number of proof-of-stake protocols, Celo Gold used for voting is never at risk. The actions of the validator groups or validators you vote for can cause you to receive lower or higher rewards, but the Celo Gold you locked will always be available to be unlocked in the future.
```

Reply:
why is that "most correct"? I had been standardizing it the opposite direction...

Reply:
Vitalik uses both in one document https://vitalik.ca/general/2019/11/22/progress.html 😂